### ISSUE TITLE
[Bug Report / Feature Request] Adding NBT storage support for items & "/bs read"

### ISSUE THREAD

**Issue by pinkpig3777** [Blackixx/BossShopPro#42]
Currently, BossShopPro does not appear to store NBT data for items. Please add that so we can add full support for custom items from other plugins in our BossShopPro GUIs/Shops. Thanks!

**Comment by mrcoffee1026** [comment]
Yes it did USED to work well enough, but from 1.15+ this feature has been horribly lacking.  Additional NBT data has been added into minecraft independently of other plugins which have been updated to make use of them, while they have updated, this plugin has not and so items are no longer matched against that data.  Items created by BossShop using the "/bs read" information are not compatible with the other plugins like they are supposed to be... and matches do not accurately match these items when they are being used as trade within BossShop.  This is a critical feature that needs to be address before this plugin can be touted to "support 1.16", it doesn't even support 1.15 at this point.
